recipes/uwsift/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "uwsift" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 23c06d3700e3f478f902c01dcbc53d45683c689bc87c289b83c9f8e1afae3f93
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - appdirs
+    - donfig
+    - h5py
+    - imageio
+    - imageio-ffmpeg
+    - matplotlib
+    - netcdf4
+    - numba
+    - numpy
+    - pillow
+    - pygrib  # [not win]
+    - pyproj
+    - pyshp
+    - pyqt
```

Acc. to https://github.com/ssec/sift/blob/1.1.0/setup.py#L192-L197:
```suggestion
    - pyqt5 >=5.9
```